django/test/testcases.py

```diff
@@ -1262,6 +1262,8 @@ def run(self):
         except Exception as e:
             self.error = e
             self.is_ready.set()
+        finally:
+            connections.close_all()
```

Comment:
Here all non-overriden connections should be closed. See the `self.connections_override` block above.